### PR TITLE
[SIE-122] refactor Experiment page to include Qualtrics and additional parameters

### PR DESCRIPTION
**Pull request by alecherryy**
I am working on this [ticket](https://cs6510.atlassian.net/browse/SIE-122). There are a lot of moving pieces, so I am breaking the work out into multiple PRs to avoid an overwhelming amount of file changes. This PR only includes refactoring of the page structure and some extra components to clean out some files.

### CODE REVIEW

File: src/stories/components/ImageSelectionTask/styles.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import '../../../scss/variables.scss';
+
+.consent-form__list {
+  @include list-clean();
+
+  li {
+    margin-bottom: $spacing-3;
+  }
+}
```
Comment (cmutuc25) [inline src/stories/components/ImageSelectionTask/styles.scss:9]: linter might get mad at this

File: src/stories/pages/Experiment/Experiment.js [hunk 2]
```diff
@@ -41,106 +51,55 @@ export const Experiment = ({ title }) => {
...
-              <p>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens
-                mnesarchum, ei periculis adipiscing per, probo populo nec ad.
-              </p>
+              <Section titleEl={HEADING} title='Consent Form'>
+                <p>Please, complete the form below before completing the
+                  study.</p>
+                { consentForms.map((form, i) => {
```
Comment (cmutuc25) [inline src/stories/pages/Experiment/Experiment.js:63]: Just have a question here.

Does this mean there are multiple consent forms as this is a map?

**Review by cmutuc25 — APPROVED** [review]

File: src/stories/components/ImageSelectionTask/styles.scss (hunk 1 above)
Comment (cmutuc25) [reply src/stories/components/ImageSelectionTask/styles.scss:9]: nvm i guess it didnt, resolving

File: src/stories/pages/Experiment/Experiment.js (hunk 2 above)
Comment (alecherryy) [reply src/stories/pages/Experiment/Experiment.js:63]: Yes!

**Pull request merged**